server/src/main/java/org/apache/iotdb/db/mpp/plan/execution/config/sys/quota/ShowSpaceQuotaTask.java

```diff
@@ -75,12 +75,13 @@ public static void buildTSBlock(TSpaceQuotaResp resp, SettableFuture<ConfigTaskR
                   Binary.valueOf(
                       spaceQuotaEntry.getValue().getDiskSize() == 0
                           ? IoTDBConstant.QUOTA_UNLIMITED
-                          : spaceQuotaEntry.getValue().getDiskSize() + "M"));
+                          : spaceQuotaEntry.getValue().getDiskSize() / 1024.0 + "G"));
```

Comment:
Suggest to use static const rather than hard code string value here

Reply:
done